.github/actions/release-notes/action.yaml

```diff
@@ -0,0 +1,123 @@
+name: "Generate and publish release notes"
+description: 'Generate and publish release notes'
+inputs:
+  githubRepository:
+    description: "Name of repository which release notes should be generated and published in 'owner/repository' format"
+    required: true
+  githubRef:
+    description: "The fully-formed ref of the branch or tag for publishing a release"
+    required: true
+  githubToken:
+    description: "Token with access to pull permissions to scylladb/scylla-operator and release permissions to repository which release is published"
+    required: true
+  goVersion:
+    description: "Version of Go"
+    required: true
+  genReleaseNotesVersionRef:
+    description: "Version ref of gen-release-notes"
+    default: master
```

Comment:
can we make this required instead? I'd like to avoid called accidentally using a rolling ref that breaks CI

Reply:
done